documentation/contributing/templates/template-procedure.adoc

```diff
@@ -19,7 +19,7 @@ _Avoid using the word 'Configuring' over and over.  Other words might include:_
 * _Customizing_
 * _Or instead of using synonyms (which becomes obvious) 'verb-alize' a word and refactor the heading - limiting (instead of 'setting resource limits')_
 
-_The standard for all Red Hat documentation is title case for all headings and titles._
+_The standard for all headings and titles._
```

Comment:
Should this be something like:

> The standard is title case for all headings and titles

I'm not sure how to parse the new sentence other than as advertising copy :-)